src/GraphQL.MicrosoftDI/ScopedAttribute.cs

```diff
@@ -0,0 +1,21 @@
+using GraphQL.MicrosoftDI;
+using GraphQL.Types;
+
+namespace GraphQL
+{
+    /// <summary>
+    /// Creates a dedicated service scope during the field resolver's execution.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method)]
+    public class ScopedAttribute : GraphQLAttribute
```

Comment:
Can I use [Scoped] with [FromServices]. If yes does the order matter?

Reply:
Scoped is applied to the method. FromServices is applied to the method argument. 

Certainly; that is the point. I think it is shown correctly in the docs? I can’t remember.

Reply:
OK.